examples/glob/src/main.rs

```diff
@@ -13,40 +13,58 @@ fn main() {
     let mut settings = Config::default();
     settings
         // File::with_name(..) is shorthand for File::from(Path::new(..))
-        .merge(File::with_name("conf/00-default.toml")).unwrap()
-        .merge(File::from(Path::new("conf/05-some.yml"))).unwrap()
-        .merge(File::from(Path::new("conf/99-extra.json"))).unwrap();
+        .merge(File::with_name("conf/00-default.toml"))
+        .unwrap()
+        .merge(File::from(Path::new("conf/01.json")))
+        .unwrap()
+        .merge(File::from(Path::new("conf/99-extra.toml")))
+        .unwrap();
```

Comment:
Why did you change this?

Reply:
Well, I just want to make it more complete, so I added files including all format we supported.
- 05-some.yml -> 01.json: I'm badly at naming, so I just named them all to [0-9].ext,  e.q. 04.ini...
- 99-extra.json -> 99-extra.toml: just for showing how json file works, so I chose 00-default to overwrite.